test/integration/install_test.go

```diff
@@ -509,7 +508,6 @@ func helmOverridesEdge(root *tls.CA) ([]string, []string) {
 	}
 	vizArgs := []string{
 		"--set", "linkerdVersion=" + TestHelper.GetVersion(),
-		"--set", "namespace=" + TestHelper.GetVizNamespace(),
```

Comment:
These are `vizArgs` right? and we aren't removing that in this PR right? :thinking:

Reply:
Correct, that's addressed in #6638